elmslie-plugin/src/main/resources/META-INF/plugin.xml

```diff
@@ -15,21 +27,14 @@
     <depends>org.jetbrains.android</depends>
 
     <actions>
-        <group id="vivid.money.elmslie.plugin.group" text="Elmslie" description="Create presentation layer">
+        <group id="vivid.money.elmslie.plugin.group" text="Elmslie" description="Generate store" popup="true">
             <separator/>
             <action id="vivid.money.elmslie.plugin.presentation.PresentationLayerAction"
                     class="vivid.money.elmslie.plugin.presentation.PresentationLayerAction"
-                    text="Generate Presentation Layer">
+                    text="Generate Presentation Layer"
+                    description="Generate classes for presentation layer">
             </action>
-            <add-to-group group-id="NewGroup" anchor="last"/>
+            <add-to-group group-id="ToolsMenu" anchor="after" relative-to-action="org.intellij.sdk.action.PopupDialogAction"/>
         </group>
     </actions>
-
-    <project-components>
-        <component>
-            <implementation-class>
-                vivid.money.elmslie.plugin.presentation.controller.PresentationLayerController
```

Comment:
Why did remove it?

Reply:
PresentationLayerController doesn't implement ProjectComponent so i guess it could be instantiated directly